README.md

```diff
@@ -30,13 +30,15 @@ In order to set up a City Scrapers project for your area you'll need a GitHub ac
 
    If you want to use a version other than 3.8 (3.6 and above are supported), you can change the version for the `--python` flag.
 
-5. Decide whether you want to output static files to AWS S3 or Microsoft Azure Blob Storage, and update the `city-scrapers-core` package with the necessary extras:
+5. Decide whether you want to output static files to AWS S3, Microsoft Azure Blob Storage, or Google Cloud Storage, and update the `city-scrapers-core` package with the necessary extras:
 
    ```shell
    # To use AWS S3
    pipenv install 'city-scrapers-core[aws]'
    # To use Microsoft Azure
-   pipenv install 'city-scrapers-core[azure]`
+   pipenv install 'city-scrapers-core[azure]'
+   # To use Google Cloud Storage
+   pipenv install 'city-scrapers-core[gcs]'
    ```
 
    Once you've updated `city-scrapers-core`, you'll need to update [`./city_scrapers/settings/prod.py`](./city_scrapers/settings/prod.py) by uncommenting the extension and storages related to your platform.
```

Comment:
I think this is in the line below this, but right now we refer to City Bureau setting up an account for people on "either platform", and we might want to change that to "S3 or Azure" because we don't have a GCP account

Reply:
Agreed, updated.